backend/data_tools/data/user_data.json5

```diff
@@ -218,6 +216,12 @@
         "GET_UPLOAD_DOCUMENT",
         "POST_UPLOAD_DOCUMENT"
       ]
+    },
+    { // 5 User Admin
+      name: "USER_ADMIN",
+      permissions: [
+        "POST_USER",
+      ],
```

Comment:
@johndeange so..... why not additional roles here?  I saw a few places in the code doing checks for `is_user_admin` on modifying or creating a user which seems to just be checking if a user has a role but it's not actually a check of the permission....    Is that by design?  I recognize there's a high chance of me asking out ignorance and/or out of context.

Reply:
Ah, thank you. I'm better informed now :-)